config/deploy/staging.rb

```diff
@@ -4,8 +4,9 @@
 set :rails_env, 'staging'
 set :branch, ENV['BRANCH'] || 'main'
 
-server 'catalog-staging1', user: 'deploy', roles: %i[web app db worker mailcatcher cron_db]
+# server 'catalog-staging1', user: 'deploy', roles: %i[web app db worker mailcatcher cron_db]
 server 'catalog-staging2', user: 'deploy', roles: %i[web app worker mailcatcher]
+server 'catalog-indexer-staging1', user: 'deploy', roles: %i[cron_prod1 cron_db worker]
```

Comment:
same question as the one in the princeton ansible pr. Can we name the staging indexer catalog-indexer-staging2 ?

Reply:
but we don't have 2 indexers.